### DISCUSSION TITLE
Broadlink e firmware v62092

### DISCUSSION THREAD

**Discussion by reiner348** [indomus/forum#1762] [Hardware & Firmware]
La mia odissea con i broadlink prosegue...
Dopo il problema con la lunghezza password wifi, ora ne è spuntato un altro con il firmware dei RM4 mini, e forse anche i pro...
Premetto che ho un RM4 mini, con firmware v52092, perfettamente integrato grazie alla guida qui di indomus. Inoltre aggiungo che è vero quanto scritto da un altro utente, anche se il broadlink viene completamente configurato nell'app, basta togliere la spunta del blocco nelle sue impostazioni (da dentro l'app) per poterlo usare tranquillamente con home assistant.
Bene... Amazon prime days, ho comprato altri 2 RM4 mini! Così finalmente avrei potuto integrare gli altri due split del condizionatore. E invece no... Stessa identica procedura di configurazione, ma quando creo la integration su HA mi dice device not supported... Unica differenza tra questi RM4 e quello già in funzione, il firmware! v52092 quello in funzione, v62092 quelli nuovi. 
Ho provato a cercare su internet e sembra che il nuovo rm4 mini 0x5216 (il numero che si vede nel log dopo l'errore di device not supported) sia supportato dalla versione python-broadlink 0.18.2, che è quella che ho installato, quindi non so proprio dove sbattere la testa...

**Comment by KC-inDomus** [comment]
https://github.com/home-assistant/core/issues/75038

**Reply by reiner348** [reply]
Il link spiega come aggiungere il device 0x5216 al python broadlink, però io ho già la versione 0.18.2 dove è presente (ho controllato). Grazie comunque! Meglio tenersi buona la spiegazione per eventuali nuovi device!
Alla fine ho risolto il problema aggiornando HA a 2022.7.5, ero fermo per pigrizia alla 2022.5... Come ho aggiornato (con un piccolo ostacolo dovuto al dover togliere dalla stringa di avvio del docker "--init" altrimenti andava continuamente in restart) ha funzionato al primo colpo.
Via, verso il prossimo problema con i broadlink! 😂

**Reply by KC-inDomus** [reply]
si spera finiscano qui :D ottimo dai
